### ISSUE TITLE
MacOS cali-query output broken

### ISSUE THREAD

**Issue by DavidPoliakoff** [LLNL/Caliper#67]
Cali file attached as txt to placate Github:

I'm trying to get JSON formatted cali-query outputs, and get correct results on Linux, incorrect on MacOS. 

```
qyburn:cali_interactive_explorer poliakoff1$ cali-query -q "SELECT time.inclusive.duration format json()" *.cali
,
{"time.inclusive.duration":5001708},
{"time.inclusive.duration":5001878}
]
```

Interestingly

```
qyburn:cali_interactive_explorer poliakoff1$ cali-query -q "SELECT * format json()" *.cali
```

Works fine, and it all works on Linux, and the table outputter is fine on both operating systems

[171110-133459_16234_soqJ6T0qUh3E.cali.txt](https://github.com/LLNL/Caliper/files/1462947/171110-133459_16234_soqJ6T0qUh3E.cali.txt)

**Comment by DavidPoliakoff** [comment]
Closing, resolved

**Comment by DavidPoliakoff** [comment]
Reopening, new bug I found turns out to be same bug

One of the files 

[171110-145327_24849_dpKGmaq0c4aM.cali.txt](https://github.com/LLNL/Caliper/files/1467813/171110-145327_24849_dpKGmaq0c4aM.cali.txt)


```
qyburn:cali_interactive_explorer poliakoff1$ cali-query -q "SELECT * FORMAT JSON()" --list-attributes *.cali
[
{"cali.attribute.prop":"84","cali.attribute.type":"string","cali.attribute.name":"event.begin#program","attribute.id":12}[
{"cali.attribute.prop":"84","cali.attribute.type":"string","cali.attribute.name":"event.begin#program","attribute.id":12}[
{"cali.attribute.prop":"84","cali.attribute.type":"string","cali.attribute.name":"event.begin#program","attribute.id":12},
{"cali.attribute.prop":"84","cali.attribute.type":"uint","cali.attribute.name":"cali.event.begin","attribute.id":16},
{"cali.attribute.prop":"84","cali.attribute.type":"uint","cali.attribute.name":"cali.event.begin","attribute.id":16},
{"cali.attribute.prop":"84","cali.attribute.type":"uint","cali.attribute.name":"cali.event.begin","attribute.id":16},
{"cali.attribute.prop":"212","cali.attribute.type":"uint","cali.attribute.name":"cali.event.attr.level","attribute.id":18},
{"cali.attribute.prop":"212","cali.attribute.type":"uint","cali.attribute.name":"cali.event.attr.level","attribute.id":18},
{"cali.attribute.prop":"212","cali.attribute.type":"uint","cali.attribute.name":"cali.event.attr.level","attribute.id":18},
{"cali.attribute.prop":"12","cali.attribute.type":"string","cali.attribute.name":"cali.caliper.version","attribute.id":23},
{"cali.attribute.prop":"12","cali.attribute.type":"string","cali.attribute.name":"cali.caliper.version","attribute.id":23},
{"cali.attribute.prop":"12","cali.attribute.type":"string","cali.attribute.name":"cali.caliper.version","attribute.id":23},
{"cali.attribute.prop":"84","cali.attribute.type":"int","cali.attribute.name":"event.begin#run_size","attribute.id":26},
{"cali.attribute.prop":"84","cali.attribute.type":"int","cali.attribute.name":"event.begin#run_size","attribute.id":26},
{"cali.attribute.prop":"84","cali.attribute.type":"int","cali.attribute.name":"event.begin#run_size","attribute.id":26},
{"cali.attribute.prop":"212","cali.attribute.type":"usr","cali.attribute.name":"cali.event.attr.ids","attribute.id":31},
{"cali.attribute.prop":"212","cali.attribute.type":"usr","cali.attribute.name":"cali.event.attr.ids","attribute.id":31},
{"cali.attribute.prop":"212","cali.attribute.type":"usr","cali.attribute.name":"cali.event.attr.ids","attribute.id":31},
{"cali.attribute.prop":"20","cali.attribute.type":"string","cali.attribute.name":"program","attribute.id":35},
{"cali.attribute.prop":"20","cali.attribute.type":"string","cali.attribute.name":"program","attribute.id":35},
{"cali.attribute.prop":"20","cali.attribute.type":"string","cali.attribute.name":"program","attribute.id":35},
{"cali.attribute.prop":"84","cali.attribute.type":"int","cali.attribute.name":"event.end#run_size","attribute.id":37},
{"cali.attribute.prop":"84","cali.attribute.type":"int","cali.attribute.name":"event.end#run_size","attribute.id":37},
{"cali.attribute.prop":"84","cali.attribute.type":"uint","cali.attribute.name":"cali.event.end","attribute.id":38},
{"cali.attribute.prop":"84","cali.attribute.type":"int","cali.attribute.name":"event.end#run_size","attribute.id":37},
{"cali.attribute.prop":"84","cali.attribute.type":"uint","cali.attribute.name":"cali.event.end","attribute.id":38},
{"cali.attribute.prop":"84","cali.attribute.type":"uint","cali.attribute.name":"cali.event.end","attribute.id":38},
{"cali.attribute.prop":"20","cali.attribute.type":"int","cali.attribute.name":"run_size","attribute.id":44},
{"cali.attribute.prop":"20","cali.attribute.type":"int","cali.attribute.name":"run_size","attribute.id":44},
{"cali.attribute.prop":"20","cali.attribute.type":"int","cali.attribute.name":"run_size","attribute.id":44},
{"cali.attribute.prop":"84","cali.attribute.type":"string","cali.attribute.name":"time.unit","attribute.id":48},
{"cali.attribute.prop":"84","cali.attribute.type":"string","cali.attribute.name":"time.unit","attribute.id":48},
{"cali.attribute.prop":"84","cali.attribute.type":"string","cali.attribute.name":"time.unit","attribute.id":48},
{"cali.attribute.prop":"84","cali.attribute.type":"bool","cali.attribute.name":"class.aggregatable","attribute.id":51},
{"cali.attribute.prop":"84","cali.attribute.type":"bool","cali.attribute.name":"class.aggregatable","attribute.id":51},
{"cali.attribute.prop":"84","cali.attribute.type":"bool","cali.attribute.name":"class.aggregatable","attribute.id":51},
{"class.aggregatable":"true","time.unit":"usec","cali.attribute.prop":"85","cali.attribute.type":"uint","cali.attribute.name":"time.inclusive.duration","attribute.id":55},
{"class.aggregatable":"true","time.unit":"usec","cali.attribute.prop":"85","cali.attribute.type":"uint","cali.attribute.name":"time.inclusive.duration","attribute.id":55},
{"cali.attribute.prop":"84","cali.attribute.type":"string","cali.attribute.name":"event.end#program","attribute.id":56},
{"class.aggregatable":"true","time.unit":"usec","cali.attribute.prop":"85","cali.attribute.type":"uint","cali.attribute.name":"time.inclusive.duration","attribute.id":55},
{"cali.attribute.prop":"84","cali.attribute.type":"string","cali.attribute.name":"event.end#program","attribute.id":56},
{"cali.attribute.prop":"84","cali.attribute.type":"string","cali.attribute.name":"event.end#program","attribute.id":56}
]
```

**Comment by alfredo-gimenez** [comment]
@daboehme reminder to close this

**Comment by daboehme** [comment]
Fixed in 86c126682c1d7f5cf30fe0eb3f5a277187abdc93
